run.sh

```diff
@@ -135,12 +135,10 @@ kubectl get pods,taskruns,pipelineruns
 # Add a secret in the physical cluster so that the event listener and interceptors can query KCP API
 cp ./.kcp/admin.kubeconfig ./.kcp/remote.kubeconfig
 gsed -i "s/\[::1\]/host.docker.internal/" ./.kcp/remote.kubeconfig
```

Comment:
gsed does not work on fedora. Had to change it to sed.

Reply:
Ahhh. OSX users 😝

Reply:
yes sorry. If someone knows how to make it transparent, I am happy to configure my machine. :D

Reply:
@chmouel ^^

Reply:
i think if you prepend the path in your .zshrc with the gnubin paths from brew and it should works: 

at least it does for me, ie i have this :

```
path=(/opt/homebrew/opt/*/libexec/gnubin
      /opt/homebrew/opt/file-formula/bin
      /opt/homebrew/sbin
      /opt/homebrew/bin $path) #prepend
```

Reply:
Fixed, thanks :)